addons/campsite/helper/fn_isInsideCampingArea.sqf

```diff
@@ -0,0 +1,41 @@
+#include "script_component.hpp"
+/*
+ * Author: Gilles
+ *
+ * TODO DOCSTRING
+ *
+ * Arguments:
+ * 0: xyz
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ *
+ * Public: No
+ *
+ * Scope: Client
+ */
+
+// Maybe consider checking this variable 1 time each session?
+scopeName "main";
```

Comment:
I was trying with ``breakOut`` as it may be cheapier in performace to just break from the loop once I found what I was looking for. I'll clean my spaghetti, sorry.